tests/unit/adapters/stripe-account-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('adapter:stripe-account', 'Unit | Adapter | stripe account', {
+  // Specify the other units that are required for this test.
+  // needs: ['serializer:foo']
+});
+
+// Replace this with your real tests.
+test('it exists', function(assert) {
```

Comment:
Ideally I would love to find a way to actually test that it does not coalesce find requests.